### ISSUE TITLE
Kodi user and home

### ISSUE THREAD

**Issue by pgcudahy** [Wintermute0110/KodiInstall#1]
Is there a reason that the scripts have to assume a kodi user with home directory of /home/kodi? Would a generic $HOME variable work instead?

**Comment by Wintermute0110** [comment]
Not really. The user name is only needed to determine the directories prefix. I'm in the process to add a configuration file like in the [MAME-Install] repository, where you can use $HOME if you wish.

**Comment by Wintermute0110** [comment]
@pgcudahy I have updated the scripts and now the read `configuration.sh`, where you can set the source code dir, build dir and installation dir as you wish. I have to update the docs to reflect this change.

EDIT Once I update the docs I will close the issue.

**Comment by Wintermute0110** [comment]
@pgcudahy This issue should be fixed now, all scripts honor `configuration.sh`. However there is a problem with the binary addons when compiling Kodi Matrix, I'm investigating it.
